app/institutions/dashboard/-components/projects-panel/component.ts

```diff
@@ -0,0 +1,39 @@
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import InstitutionModel from 'ember-osf-web/models/institution';
+
+export default class ProjectsPanel extends Component {
+    institution!: InstitutionModel;
+
+    chartOptions = {
+        aspectRatio: 1,
+        legend: {
+            display: false,
+        },
+    };
+
+    @computed('institution.statSummary.{numPrivateProjects,numPublicProjects}')
+    get numProjects() {
+        if (!this.institution) {
+            return 0;
+        }
+        return this.institution.statSummary.numPrivateProjects + this.institution.statSummary.numPublicProjects;
+    }
+
+    @computed('institution.statSummary.{numPrivateProjects,numPublicProjects}')
+    get chartData() {
+        return {
+            labels: ['Public', 'Private'],
+            datasets: [{
+                data: [
+                    this.institution.statSummary.numPublicProjects,
+                    this.institution.statSummary.numPrivateProjects,
```

Comment:
Consider destructuring these into constants for readability;

Reply:
Actually, even better would be to use the `alias` decorator then you could use them here and in the template:
```ts
@alias('this.institution.statSummary.numPublicProjects') numPublicProjects?: number;
@alias('this.institution.statSummary.numPrivateProjects') numPrivateProjects?: number;
```